### DISCUSSION TITLE
the battery status colors are not displayed

### DISCUSSION THREAD

**Discussion by Malvu** [andrew-codechimp/HA-Battery-Notes#1617] [General]
Hello to you

I love the principle of your HACS, and installed it following the tutorial from Smart Home Australia, I should point out that I only understand English a little.
I find myself with a little problem.
the battery status colors are not displayed when creating maps

**Comment by andrew-codechimp** [comment]
I have success with this way of doing it

https://andrew-codechimp.github.io/HA-Battery-Notes/community/#battery-state-card

**Reply by Malvu** [reply]
Good morning

Should I include it alongside Battery note?

If yes, how ?

**Reply by andrew-codechimp** [reply]
Yes, you will need to install it via HACS, just search for Battery State Card.  Then you can include the YAML to create the card on a dashboard.
